jenkins/opensearch/integ-test.jenkinsfile

```diff
@@ -80,66 +96,78 @@ pipeline {
-                                                echo "Downloading from S3: ${artifactPath}"
-                                                downloadFromS3(
-                                                    destPath: "$WORKSPACE/artifacts",
-                                                    bucket: "${ARTIFACT_BUCKET_NAME}",
-                                                    path: "${artifactPath}/",
-                                                    force: true
-                                                )
-                                                sh("mv -v $WORKSPACE/artifacts/${artifactPath} $WORKSPACE")
+                                                sh("echo ${local_component} with index ${local_component_index} will sleep ${wait_seconds} seconds to reduce load && sleep ${wait_seconds}")
+                                                unstash "opensearch-build-repo-$BUILD_NUMBER"
+                                                sh("rm -rf test-results")
                                                 runIntegTestScript(
-                                                    jobName: BUILD_JOB_NAME,
+                                                    jobName: "$BUILD_JOB_NAME",
                                                     componentName: "${local_component}",
-                                                    buildManifest: BUILD_MANIFEST,
+                                                    buildManifest: "$BUILD_MANIFEST",
                                                     testManifest: "manifests/${TEST_MANIFEST}",
-                                                    localPath: "${WORKSPACE}/${distribution}"
+                                                    localPath: "${WORKSPACE}/${distribution}",
+                                                    switchUserNonRoot: "${switch_user_non_root}"
                                                 )
                                             }
                                         } catch (e) {
                                             echo "Error running integtest for component ${local_component}"
                                             throw e
                                         } finally {
                                             echo "Completed running integtest for component ${local_component}"
-                                            uploadTestResults(
-                                                buildManifestFileName: BUILD_MANIFEST,
-                                                jobName: JOB_NAME
-                                            )
+                                            //uploadTestResults(
+                                            //    buildManifestFileName: BUILD_MANIFEST,
+                                            //    jobName: JOB_NAME
+                                            //)
```

Comment:
Why we don't upload the test results here?

Reply:
uncomment now.